07-modeling.Rmd

```diff
@@ -711,7 +709,7 @@ tidy(log_biden_int) %>%
   print_gt_book(knitr::opts_current$get()[["label"]])
 ```
 
-The results from the interaction model show that the interaction between early voting behavior and gender is significant.  To better understand what this interaction means, we will want to plot the predicted probabilities with an interaction plot.  Let's first obtain the predicted probabilities for each possible combination of variables using the `augment()` function.
+The results from the interaction model show that the interaction between early voting behavior and gender is significant.  To better understand what this interaction means, we can plot the predicted probabilities with an interaction plot.  Let's first obtain the predicted probabilities for each possible combination of variables using the `augment()` function.
```

Comment:
Need to find a different example to illustrate interaction plots since this one is no longer significant!